operator/v1/types_network.go

```diff
@@ -137,6 +137,7 @@ type NetworkSpec struct {
 	// respective documentation and configuration options.
 	// +openshift:enable:FeatureGate=AdditionalRoutingCapabilities
 	// +optional
+	// +nullable
```

Comment:
I did some more research and I think this is not the correct approach, please remove

I will override the no new required fields, as I think it is erroneous

Reply:
will do!